<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>serial</class>
 <widget class="QMainWindow" name="serial">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>883</width>
    <height>592</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>serial</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab1">
       <attribute name="title">
        <string>遥控</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QGroupBox" name="groupBox">
            <property name="title">
             <string>配置区</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_2">
               <item>
                <widget class="QLabel" name="statusLab">
                 <property name="minimumSize">
                  <size>
                   <width>110</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Disconneted</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QGroupBox" name="groupBox_3">
                 <property name="title">
                  <string/>
                 </property>
                 <layout class="QGridLayout" name="gridLayout">
                  <item row="0" column="1">
                   <widget class="QComboBox" name="serialBox"/>
                  </item>
                  <item row="1" column="0">
                   <widget class="QLabel" name="baudratelabel">
                    <property name="text">
                     <string>波特率</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="1">
                   <widget class="QComboBox" name="baudrateBox">
                    <item>
                     <property name="text">
                      <string>9600</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>115200</string>
                     </property>
                    </item>
                   </widget>
                  </item>
                  <item row="0" column="0">
                   <widget class="QLabel" name="comlabel">
                    <property name="text">
                     <string>端口号</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="openButton">
                 <property name="text">
                  <string>打开串口</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="closeButton">
                 <property name="text">
                  <string>关闭串口</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="hexDisplayBox">
                 <property name="text">
                  <string>十六进制显示</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="hexSendBox">
                 <property name="text">
                  <string>十六进制发送</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="clearButton">
                 <property name="text">
                  <string>清空接收区</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="verticalSpacer">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_2">
            <property name="title">
             <string/>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QLineEdit" name="sendData"/>
               </item>
               <item>
                <widget class="QPushButton" name="sendButton">
                 <property name="text">
                  <string>Send</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QGroupBox" name="groupBox_5">
               <property name="title">
                <string>接收区</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_5">
                <item>
                 <widget class="QPlainTextEdit" name="receiveData"/>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="groupBox_4">
               <property name="title">
                <string>控制区</string>
               </property>
               <layout class="QGridLayout" name="gridLayout_3">
                <item row="2" column="0">
                 <layout class="QHBoxLayout" name="horizontalLayout_4">
                  <item>
                   <widget class="QLabel" name="label_4">
                    <property name="text">
                     <string>当前速度：</string>
                    </property>
                    <property name="alignment">
                     <set>Qt::AlignCenter</set>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="speedLab">
                    <property name="text">
                     <string>0</string>
                    </property>
                    <property name="alignment">
                     <set>Qt::AlignCenter</set>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item row="2" column="2">
                 <layout class="QHBoxLayout" name="horizontalLayout_5">
                  <item>
                   <widget class="QLabel" name="label_5">
                    <property name="text">
                     <string>当前角速度：</string>
                    </property>
                    <property name="alignment">
                     <set>Qt::AlignCenter</set>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="angularSpeedLab">
                    <property name="text">
                     <string>0</string>
                    </property>
                    <property name="alignment">
                     <set>Qt::AlignCenter</set>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item row="2" column="1">
                 <widget class="QPushButton" name="stopButton">
                  <property name="text">
                   <string>stop</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="2">
                 <widget class="QDial" name="angularSpeedDial"/>
                </item>
                <item row="0" column="0">
                 <widget class="QDial" name="speedDial"/>
                </item>
                <item row="0" column="1">
                 <layout class="QVBoxLayout" name="verticalLayout_6">
                  <item>
                   <widget class="QPushButton" name="modeButton">
                    <property name="text">
                     <string>模式切换</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_3">
                    <item>
                     <widget class="QLabel" name="label_2">
                      <property name="text">
                       <string>ModeStatus:</string>
                      </property>
                      <property name="alignment">
                       <set>Qt::AlignCenter</set>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLabel" name="modeLab">
                      <property name="text">
                       <string>手动</string>
                      </property>
                      <property name="alignment">
                       <set>Qt::AlignCenter</set>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </item>
                <item row="3" column="1">
                 <widget class="QPushButton" name="pushButton">
                  <property name="text">
                   <string>restore</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>883</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menufiles">
    <property name="title">
     <string>files</string>
    </property>
    <addaction name="actionsave"/>
   </widget>
   <widget class="QMenu" name="menutools">
    <property name="title">
     <string>tools</string>
    </property>
    <addaction name="actioncopy"/>
   </widget>
   <addaction name="menufiles"/>
   <addaction name="menutools"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionsave">
   <property name="text">
    <string>save</string>
   </property>
  </action>
  <action name="actioncopy">
   <property name="text">
    <string>copy</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
